ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml

```diff
@@ -369,4 +402,88 @@ Popup {
             onCloseActivityCenter: root.close()
         }
     }
+
+    Component {
+        id: pairDeviceDialog
+
+        StatusDialog {
```

Comment:
When does this get destroyed? You should either set `destroyOnClose: true` or `destroy()` it manually in the `onClosed` handler